packages/app/src/index.ts

```diff
@@ -0,0 +1,17 @@
+import { AeroGearConfiguration, ConfigurationHelper } from "@aerogear/core";
+
+declare var window: any;
+
+let config: ConfigurationHelper;
+
+export function init(configuration: AeroGearConfiguration) {
+  config = new ConfigurationHelper(configuration);
+  window.Metrics.init(config);
+}
+
+export function getAuth() {
```

Comment:
Going to check if we can still reference actual types using `///` notation so users will benefit from types.